File: codeforces/60.holiday_of_equality/holiday_of_equality.py
# Author: José Rodolfo (jric2002)
n = int(input())
a = list(map(int, input().split(" ")))
total_welfare = 0
max_welfare = 0
for i in range(0, n):
  if (a[i] > max_welfare):
    max_welfare = a[i]
  total_welfare += a[i]
s = (n * max_welfare) - total_welfare
print(s)

File: codeforces/low_difficulty/800_points/60.holiday_of_equality_[758A]/holiday_of_equality.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n;
  unsigned int a, s, max_welfare = 0, total_welfare = 0;
  cin >> n;
  for (unsigned short int i = 0; i < n; i++) {
    cin >> a;
    if (a > max_welfare) {
      max_welfare = a;
    }
    total_welfare += a;
  }
  s = (n * max_welfare) - total_welfare;
  cout << s << endl;
  return 0;
}